processing file asmFiles/assoc.asm (1 of 38)
processing file asmFiles/assoc.asm (1 of 38)
processing file asmFiles/beq1.asm (2 of 38)
processing file asmFiles/beq2.asm (3 of 38)
processing file asmFiles/bge1.asm (4 of 38)
processing file asmFiles/bge2.asm (5 of 38)
processing file asmFiles/bgeu1.asm (6 of 38)
processing file asmFiles/bgeu2.asm (7 of 38)
processing file asmFiles/blt1.asm (8 of 38)
processing file asmFiles/blt2.asm (9 of 38)
processing file asmFiles/bltu1.asm (10 of 38)
processing file asmFiles/bltu2.asm (11 of 38)
processing file asmFiles/bne1.asm (12 of 38)
processing file asmFiles/bne2.asm (13 of 38)
processing file asmFiles/break2.asm (14 of 38)
processing file asmFiles/test.branchjump1.asm (1 of 6)
processing file asmFiles/test.branchjump2.asm (2 of 6)
processing file asmFiles/test.ldst1.asm (3 of 6)
processing file asmFiles/test.ldst2.asm (4 of 6)
processing file asmFiles/test.loadstore.asm (5 of 6)
processing file asmFiles/test.rtype.asm (6 of 6)
processing file asmFiles/mult.asm (1 of 2)
processing file asmFiles/mult_procedure.asm (2 of 2)
--- memsim.hex	2025-10-27 18:47:57.610514674 -0400
+++ memcpu.hex	2025-10-27 18:47:57.605514481 -0400
@@ -31,8 +31,8 @@
 :04001E00FFC101130A
 :04001F0001E12023B8
 :0400200000008067F5
-:040C400000000008A8
-:043FFD0000000005BB
+:040C400000000009A7
+:043FFD000000038439
 :043FFE000000001EA1
-:043FFF000000003C82
+:043FFF0000000002BC
 :00000001FF
processing file asmFiles/mult.asm (1 of 2)
processing file asmFiles/mult_procedure.asm (2 of 2)
--- memsim.hex	2025-10-27 18:48:24.026531323 -0400
+++ memcpu.hex	2025-10-27 18:48:24.021531130 -0400
@@ -31,8 +31,8 @@
 :04001E00FFC101130A
 :04001F0001E12023B8
 :0400200000008067F5
-:040C400000000008A8
-:043FFD0000000005BB
+:040C400000000009A7
+:043FFD000000038439
 :043FFE000000001EA1
-:043FFF000000003C82
+:043FFF0000000002BC
 :00000001FF
processing file asmFiles/count_days.asm (1 of 1)
processing file asmFiles/fib.asm (1 of 1)
--- memsim.hex	2025-10-27 18:49:03.270041651 -0400
+++ memcpu.hex	2025-10-27 18:49:03.265041459 -0400
@@ -14,28 +14,8 @@
 :04000D00FE0814E3F2
 :04000E00FFFFFFFFF2
 :0400210000000001DA
-:0400220000000001D9
+:040022000000008456
 :0400230000000002D7
-:0400240000000003D5
-:0400250000000005D2
-:0400260000000008CE
-:040027000000000DC8
-:0400280000000015BF
-:0400290000000022B1
-:04002A00000000379B
-:04002B000000005978
-:04002C000000009040
-:04002D00000000E9E6
-:04002E000000017954
-:04002F000000026269
-:04003000000003DBEE
-:040031000000063D88
-:0400320000000A18A8
-:040033000000105564
-:0400340000001A6D41
-:0400350000002AC2DB
-:040036000000452F52
-:0400370000006FF165
 :0403C0000000001623
-:040C4000000000367A
+:040C40000000002B85
 :00000001FF
processing file asmFiles/assoc.asm (1 of 37)
processing file asmFiles/beq1.asm (2 of 37)
processing file asmFiles/beq2.asm (3 of 37)
processing file asmFiles/bge1.asm (4 of 37)
processing file asmFiles/bge2.asm (5 of 37)
processing file asmFiles/bgeu1.asm (6 of 37)
processing file asmFiles/bgeu2.asm (7 of 37)
processing file asmFiles/blt1.asm (8 of 37)
processing file asmFiles/blt2.asm (9 of 37)
processing file asmFiles/bltu1.asm (10 of 37)
processing file asmFiles/bltu2.asm (11 of 37)
processing file asmFiles/bne1.asm (12 of 37)
processing file asmFiles/bne2.asm (13 of 37)
processing file asmFiles/break.asm (14 of 37)
--- memsim.hex	2025-10-27 18:51:04.428706830 -0400
+++ memcpu.hex	2025-10-27 18:51:04.424706676 -0400
@@ -29,7 +29,7 @@
 :04001C002340611309
 :04001D002020262356
 :04001E000000006F6F
-:04008000000000017B
+:04008000FFFFFEADD3
 :040081000000000279
 :040084000000000177
 :040C400000000004AC
processing file asmFiles/count_days.asm (15 of 37)
processing file asmFiles/fib.asm (16 of 37)
--- memsim.hex	2025-10-27 18:51:09.057885176 -0400
+++ memcpu.hex	2025-10-27 18:51:09.052884984 -0400
@@ -14,28 +14,8 @@
 :04000D00FE0814E3F2
 :04000E00FFFFFFFFF2
 :0400210000000001DA
-:0400220000000001D9
+:040022000000008456
 :0400230000000002D7
-:0400240000000003D5
-:0400250000000005D2
-:0400260000000008CE
-:040027000000000DC8
-:0400280000000015BF
-:0400290000000022B1
-:04002A00000000379B
-:04002B000000005978
-:04002C000000009040
-:04002D00000000E9E6
-:04002E000000017954
-:04002F000000026269
-:04003000000003DBEE
-:040031000000063D88
-:0400320000000A18A8
-:040033000000105564
-:0400340000001A6D41
-:0400350000002AC2DB
-:040036000000452F52
-:0400370000006FF165
 :0403C0000000001623
-:040C4000000000367A
+:040C40000000002B85
 :00000001FF
processing file asmFiles/halt.asm (17 of 37)
processing file asmFiles/HazardCF.asm (18 of 37)
processing file asmFiles/HazardData.asm (19 of 37)
--- memsim.hex	2025-10-27 18:51:15.988152175 -0400
+++ memcpu.hex	2025-10-27 18:51:15.983151983 -0400
@@ -13,7 +13,7 @@
 :04000C00000E04637B
 :04000D0001DE8EB3CF
 :04000E00FFFFFFFFF2
-:0403C8000000000928
+:0403C80000000F24FE
 :0403C9000000000927
 :040C400000000002AE
 :00000001FF
processing file asmFiles/HazardStr.asm (20 of 37)
--- memsim.hex	2025-10-27 18:51:18.260239711 -0400
+++ memcpu.hex	2025-10-27 18:51:18.255239518 -0400
@@ -12,8 +12,6 @@
 :0403C00011111111F5
 :0403C10022222222B0
 :0403C200333333336B
-:0403C80011111111ED
-:0403C90022222222A8
-:0403CA003333333363
+:0403C90000000F041D
 :040C400000000002AE
 :00000001FF
processing file asmFiles/ibd.asm (21 of 37)
--- memsim.hex	2025-10-27 18:51:20.582329173 -0400
+++ memcpu.hex	2025-10-27 18:51:20.577328980 -0400
@@ -29,13 +29,13 @@
 :04001C00010A2A2388
 :04001D00FFFFFFFFE3
 :04010000FFFFFBCD35
-:04010100FFFFFBCF32
+:04010100000003FCFB
 :0401100000000234B5
 :0401C000FFFFFBCD75
-:0401C100FFFFFBCE73
-:0401C200FFFFFBCC74
-:0401C300FFFFFBCF70
+:0401C1000000040135
+:0401C200000003FF37
+:0401C300000003FC39
 :0401C400FFFFFBCD71
-:0401C500FFFFFBCF6E
+:0401C500000003FC37
 :040C400000000007A9
 :00000001FF
processing file asmFiles/idSC.asm (22 of 37)
--- memsim.hex	2025-10-27 18:51:22.851416593 -0400
+++ memcpu.hex	2025-10-27 18:51:22.846416400 -0400
@@ -22,13 +22,13 @@
 :04001500009A282302
 :04001600FFFFFFFFEA
 :040140000000023485
-:040141000000023583
-:04014200000008AD04
+:0401410000000501B4
+:0401420000000B7C32
 :04015000000006782D
 :0401C0000000023405
-:0401C1000000023503
+:0401C1000000050134
 :0401C20000000678BB
-:0401C3000000023501
-:0401C400000008AD82
+:0401C3000000050132
+:0401C40000000B7CB0
 :040C400000000006AA
 :00000001FF
processing file asmFiles/ihit__dhit.asm (23 of 37)
processing file asmFiles/ihit_before_dhit.asm (1 of 2)
processing file asmFiles/ihit_before_dhit.asm (1 of 2)
processing file asmFiles/ihit_before_dhit.asm (1 of 2)
processing file asmFiles/ihit_dhit_same.asm (2 of 2)
processing file asmFiles/fib.asm (1 of 1)
--- memsim.hex	2025-10-27 19:13:11.488795584 -0400
+++ memcpu.hex	2025-10-27 19:13:11.484795429 -0400
@@ -14,28 +14,8 @@
 :04000D00FE0814E3F2
 :04000E00FFFFFFFFF2
 :0400210000000001DA
-:0400220000000001D9
+:040022000000008456
 :0400230000000002D7
-:0400240000000003D5
-:0400250000000005D2
-:0400260000000008CE
-:040027000000000DC8
-:0400280000000015BF
-:0400290000000022B1
-:04002A00000000379B
-:04002B000000005978
-:04002C000000009040
-:04002D00000000E9E6
-:04002E000000017954
-:04002F000000026269
-:04003000000003DBEE
-:040031000000063D88
-:0400320000000A18A8
-:040033000000105564
-:0400340000001A6D41
-:0400350000002AC2DB
-:040036000000452F52
-:0400370000006FF165
 :0403C0000000001623
-:040C4000000000367A
+:040C40000000002B85
 :00000001FF
processing file asmFiles/count_days.asm (1 of 1)
processing file asmFiles/mergesort.asm (1 of 1)
processing file asmFiles/mult.asm (1 of 2)
processing file asmFiles/mult_procedure.asm (2 of 2)
--- memsim.hex	2025-10-27 19:14:37.127091901 -0400
+++ memcpu.hex	2025-10-27 19:14:37.122091708 -0400
@@ -31,8 +31,8 @@
 :04001E00FFC101130A
 :04001F0001E12023B8
 :0400200000008067F5
-:040C400000000008A8
-:043FFD0000000005BB
+:040C400000000009A7
+:043FFD000000038439
 :043FFE000000001EA1
-:043FFF000000003C82
+:043FFF0000000002BC
 :00000001FF
processing file asmFiles/assoc.asm (1 of 37)
processing file asmFiles/beq1.asm (2 of 37)
processing file asmFiles/beq2.asm (3 of 37)
processing file asmFiles/bge1.asm (4 of 37)
processing file asmFiles/bge2.asm (5 of 37)
processing file asmFiles/bgeu1.asm (6 of 37)
processing file asmFiles/bgeu2.asm (7 of 37)
processing file asmFiles/blt1.asm (8 of 37)
processing file asmFiles/blt2.asm (9 of 37)
processing file asmFiles/bltu1.asm (10 of 37)
processing file asmFiles/bltu2.asm (11 of 37)
processing file asmFiles/bne1.asm (12 of 37)
processing file asmFiles/bne2.asm (13 of 37)
processing file asmFiles/break.asm (14 of 37)
--- memsim.hex	2025-10-27 19:25:21.301887525 -0400
+++ memcpu.hex	2025-10-27 19:25:21.296887332 -0400
@@ -29,7 +29,7 @@
 :04001C002340611309
 :04001D002020262356
 :04001E000000006F6F
-:04008000000000017B
+:04008000FFFFFEADD3
 :040081000000000279
 :040084000000000177
 :040C400000000004AC
processing file asmFiles/count_days.asm (15 of 37)
processing file asmFiles/fib.asm (16 of 37)
--- memsim.hex	2025-10-27 19:25:25.938065979 -0400
+++ memcpu.hex	2025-10-27 19:25:25.933065787 -0400
@@ -14,28 +14,8 @@
 :04000D00FE0814E3F2
 :04000E00FFFFFFFFF2
 :0400210000000001DA
-:0400220000000001D9
+:040022000000008456
 :0400230000000002D7
-:0400240000000003D5
-:0400250000000005D2
-:0400260000000008CE
-:040027000000000DC8
-:0400280000000015BF
-:0400290000000022B1
-:04002A00000000379B
-:04002B000000005978
-:04002C000000009040
-:04002D00000000E9E6
-:04002E000000017954
-:04002F000000026269
-:04003000000003DBEE
-:040031000000063D88
-:0400320000000A18A8
-:040033000000105564
-:0400340000001A6D41
-:0400350000002AC2DB
-:040036000000452F52
-:0400370000006FF165
 :0403C0000000001623
-:040C4000000000367A
+:040C40000000002B85
 :00000001FF
processing file asmFiles/halt.asm (17 of 37)
processing file asmFiles/HazardCF.asm (18 of 37)
processing file asmFiles/HazardData.asm (19 of 37)
--- memsim.hex	2025-10-27 19:25:32.663324845 -0400
+++ memcpu.hex	2025-10-27 19:25:32.659324692 -0400
@@ -13,7 +13,7 @@
 :04000C00000E04637B
 :04000D0001DE8EB3CF
 :04000E00FFFFFFFFF2
-:0403C8000000000928
+:0403C80000000F24FE
 :0403C9000000000927
 :040C400000000002AE
 :00000001FF
processing file asmFiles/HazardStr.asm (20 of 37)
--- memsim.hex	2025-10-27 19:25:34.932412186 -0400
+++ memcpu.hex	2025-10-27 19:25:34.927411994 -0400
@@ -12,8 +12,6 @@
 :0403C00011111111F5
 :0403C10022222222B0
 :0403C200333333336B
-:0403C80011111111ED
-:0403C90022222222A8
-:0403CA003333333363
+:0403C90000000F041D
 :040C400000000002AE
 :00000001FF
processing file asmFiles/ibd.asm (21 of 37)
--- memsim.hex	2025-10-27 19:25:37.257501683 -0400
+++ memcpu.hex	2025-10-27 19:25:37.253501529 -0400
@@ -29,13 +29,13 @@
 :04001C00010A2A2388
 :04001D00FFFFFFFFE3
 :04010000FFFFFBCD35
-:04010100FFFFFBCF32
+:04010100000003FCFB
 :0401100000000234B5
 :0401C000FFFFFBCD75
-:0401C100FFFFFBCE73
-:0401C200FFFFFBCC74
-:0401C300FFFFFBCF70
+:0401C1000000040135
+:0401C200000003FF37
+:0401C300000003FC39
 :0401C400FFFFFBCD71
-:0401C500FFFFFBCF6E
+:0401C500000003FC37
 :040C400000000007A9
 :00000001FF
processing file asmFiles/idSC.asm (22 of 37)
--- memsim.hex	2025-10-27 19:25:39.536589409 -0400
+++ memcpu.hex	2025-10-27 19:25:39.532589255 -0400
@@ -22,13 +22,13 @@
 :04001500009A282302
 :04001600FFFFFFFFEA
 :040140000000023485
-:040141000000023583
-:04014200000008AD04
+:0401410000000501B4
+:0401420000000B7C32
 :04015000000006782D
 :0401C0000000023405
-:0401C1000000023503
+:0401C1000000050134
 :0401C20000000678BB
-:0401C3000000023501
-:0401C400000008AD82
+:0401C3000000050132
+:0401C40000000B7CB0
 :040C400000000006AA
 :00000001FF
processing file asmFiles/ihit_before_dhit.asm (23 of 37)
processing file asmFiles/ihit_dhit_same.asm (24 of 37)
processing file asmFiles/jal.asm (25 of 37)
processing file asmFiles/jalr2.asm (26 of 37)
processing file asmFiles/jalr.asm (27 of 37)
processing file asmFiles/mergesort.asm (28 of 37)
processing file asmFiles/assoc.asm (1 of 37)
--- memsim.hex	2025-10-28 16:35:13.617379178 -0400
+++ memcpu.hex	2025-10-28 16:35:13.611378947 -0400
@@ -33,4 +33,5 @@
 :0401C3000000023402
 :0401C40000000678B9
 :0401C500FFFFFABC82
+:040C400000000008A8
 :00000001FF
processing file asmFiles/beq1.asm (2 of 37)
processing file asmFiles/beq2.asm (3 of 37)
processing file asmFiles/bge1.asm (4 of 37)
